chef_master/source/resource_mount.rst

```diff
@@ -41,7 +41,7 @@ The full syntax for all of the properties that are available to the **mount** re
      password                   String
      provider                   Chef::Provider::Mount
      subscribes                 # see description
-     supports                   Array
+     supports                   Hash # defaults to { :remount => false }
```

Comment:
The interesting thing about the code is that `supports` does in fact support an `Array` as well. When an `Array` is specified the resulting `Hash` are all of the keys defined in the `Array` with values set to `true`.